ups/lsst_ci.table

```diff
@@ -1,5 +1,6 @@
 setupRequired(lsst_apps)
 setupRequired(lsst_distrib)
+setupRequired(lsst_sitcom)
```

Comment:
I think we agreed that this should just be `summit_utils`, not everything in `lsst_sitcom`.